server/controllers/api/users/me.ts

```diff
@@ -72,6 +72,11 @@ meRouter.get('/me/videos/:videoId/rating',
   asyncMiddleware(getUserVideoRating)
 )
 
+meRouter.get('/me/videos/ratings',
```

Comment:
Prefer to use the account endpoint instead https://github.com/Chocobozzz/PeerTube/blob/develop/server/controllers/api/accounts.ts#L37, because in the future some users may want to show what they liked.

Reply:
I'm OK to make this change, but just to be sure: isn't playlists the way to share a set of videos to others? And isn't exposing my likes in the API a bit tricky on the privacy topic?

Reply:
> I'm OK to make this change, but just to be sure: isn't playlists the way to share a set of videos to others?

Yes you're right. But I still prefer to use the account endpoint, just in case.

> And isn't exposing my likes in the API a bit tricky on the privacy topic?

By default this endpoint should be only accessible by the authenticated user (add a validator middleware)

Reply:
>  by the authenticated user

Just to be sure: you mean the account owner, not any authenticated user?

Reply:
Yep!